### ISSUE TITLE
Kinetics 400 models

### ISSUE THREAD

**Issue by lovelyczli** [Atze00/MoViNet-pytorch#27]
Thanks for your Pytorch implementation for MoViNet.
Could you please provide a model trained on Kinetics 400 as Table 9 in [1]. 
It is quite important for our future works.
Looking forward to your reply.

[1] MoViNets: Mobile Video Networks for Efficient Video Recognition

**Comment by Hyperparticle** [comment]
Hi, the official repo now has Kinetics 400 pretrained checkpoints for base models available here (streaming models should be available soon): https://github.com/tensorflow/models/tree/master/official/projects/movinet#kinetics-400. Accuracy should be a bit better than what is reported in the paper. You should be able to convert this to PyTorch similarly to the Kinetics 600 models.
